src/test/java/com/alvarium/streams/mqtt-config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "MQTT",
```

Comment:
Note that this value in the Go SDK is lowercase. I'd like to preserve standard config if possible.

Reply:
issue #42 should address this issue when implemented.